### PR TITLE
Stop logging undefined actions

### PR DESCRIPTION
**Pull request by zmoazeni**
Some webhook events are printing like `push.undefined` which we don't need.

Others make sense like `pull_request.closed` so lets keep that

### CODE REVIEW

File: lib/github/middleware.js [hunk 1]
```diff
@@ -41,9 +41,13 @@ module.exports = function middleware (callback) {
     enhanceOctokit(context.github, context.log)
 
     newrelic.setControllerName(`github/events.${context.name}`)
+    let webhookEvent = context.name
+    if (context.payload.action !== undefined) {
+      webhookEvent = `${webhookEvent}.context.payload.action`
```
Comment (jules2689) [inline lib/github/middleware.js]:
```suggestion
      webhookEvent = `${webhookEvent}.${context.payload.action}`
```

I think you wanted this?

On another note, would it make sense to make this `Webhook Name: context.name` and `Action: context.payload.action` (if applicable)?

Comment (zmoazeni) [reply lib/github/middleware.js]: Lol yeah that should be changed.

> On another note, would it make sense to make this Webhook Name: context.name and Action: context.payload.action (if applicable)?

I thought about that. But I preferred the single line dot notation myself. It aligns with how we subscribe to events with the robot.

File: lib/github/middleware.js [hunk 2]
```diff
@@ -41,9 +41,13 @@ module.exports = function middleware (callback) {
     enhanceOctokit(context.github, context.log)
 
     newrelic.setControllerName(`github/events.${context.name}`)
+    let webhookEvent = context.name
+    if (context.payload.action !== undefined) {
```
Comment (tebriel) [inline lib/github/middleware.js]: I think this should be `!= null` in case the key exists in the dictionary but has a null value.

Comment (zmoazeni) [reply lib/github/middleware.js]: I just updated to check for truthiness instead.

**Pull request merged**